tree/testdata/TestRootStyle.golden

```diff
@@ -1,3 +1,3 @@
-[48;2;89;86;224mRoot[0m
```

Comment:
We need to keep ansi sequences in this golden file since we're testing that the root element styling is different from the other elements in the list. We also need to remove the `ansi.Strip` from the `TestRootStyle`